docs/source/reference/numpysupported.rst

```diff
@@ -627,7 +627,13 @@ The following :py:class:`Generator` methods are supported:
 * :func:`numpy.random.Generator().laplace()` (*)
 * :func:`numpy.random.Generator().logistic()` (*)
 * :func:`numpy.random.Generator().lognormal()` (*)
+* :func:`numpy.random.Generator().logseries()` (Accepts float values as well as datatypes
+ that cast to floats. Array values for p are currently not supported.)
 * :func:`numpy.random.Generator().negative_binomial()` (*)
+* :func:`numpy.random.Generator().noncentral_chisquare()` (Accepts float values as well as datatypes
+ that cast to floats. Array values for dfnum and nonc are currently not supported.)
+* :func:`numpy.random.Generator().noncentral_f()` (Accepts float values as well as datatypes
+ that cast to floats. Array values for dfnum, dfden and nonc are currently not supported.)
```

Comment:
```suggestion
* :func:`numpy.random.Generator().logseries()` (Accepts float values as well as datatypes
  that cast to floats. Array values for p are currently not supported.)
* :func:`numpy.random.Generator().negative_binomial()` (*)
* :func:`numpy.random.Generator().noncentral_chisquare()` (Accepts float values as well as datatypes
  that cast to floats. Array values for dfnum and nonc are currently not supported.)
* :func:`numpy.random.Generator().noncentral_f()` (Accepts float values as well as datatypes
  that cast to floats. Array values for dfnum, dfden and nonc are currently not supported.)
```
I think something like this ought to fix the doc build. IIRC continued item lines in bulleted lists need the same indent level as the start of the item.